src/main/java/frc/team2412/robot/commands/launcher/SetAngleCommand.java

```diff
@@ -3,26 +3,29 @@
 import edu.wpi.first.math.MathUtil;
 import edu.wpi.first.wpilibj2.command.Command;
 import frc.team2412.robot.subsystems.LauncherSubsystem;
+import java.util.function.DoubleSupplier;
 // this command adjusts the angle to a value that is input by the user.
 
 public class SetAngleCommand extends Command {
 	private final LauncherSubsystem launcherSubsystem;
-	private final double launcherAngle;
+	private final DoubleSupplier launcherAngle;
 
-	public SetAngleCommand(LauncherSubsystem launcherSubsystem, double angle) {
+	public SetAngleCommand(LauncherSubsystem launcherSubsystem, DoubleSupplier angle) {
 		this.launcherSubsystem = launcherSubsystem;
 		this.launcherAngle = angle;
 		addRequirements(launcherSubsystem);
 	}
 
 	@Override
-	public void initialize() {
-		launcherSubsystem.setAngle(launcherAngle);
+	public void execute() {
+		launcherSubsystem.setAngle(launcherAngle.getAsDouble());
```

Comment:
Consider keeping this as `initialize()` to avoid callers having to check if the command has ended before they update the value in the DoubleSupplier (to avoid bugs where they don't handle the command ending since it reached the target angle).